### DISCUSSION TITLE
Adding instanced customizable guild houses

### DISCUSSION THREAD

**Discussion by GoblinKing1337** [chromiecraft/chromiecraft#6612] [Suggestions]
Any chance we could implement some form of instanced guild houses? Some kind of module already exists for AC: https://github.com/azerothcore/mod-guildhouse although I have no idea how fleshed out it is, nor do I have any experience with this stuff, but I think it'd be a pretty neat way to customize the few guilds on CC.

Maybe restrict to certain guild sizes or guilds that have accomplished certain things before you can have an instanced house.

**Comment by UnePomme** [comment]
As long as it wouldnt take population away from the major cities, I would love to see something like this.
